bundles/org.openhab.ui/web/src/components/widgets/widget-mixin.js

```diff
@@ -163,6 +163,7 @@ export default {
           this.$set(this.widgetVars, varKey, this.context.vars[varKey])
         }
       }
+      Object.assign(widget.slots, this.context.component.slots)
```

Comment:
This breaks custom widget loading elsewhere (I have the issue on canvas layers). I did this to allow custom widgets showing in the right part of the accordion trigger. @ghys do you have an idea on how to achieve this without breaking stuff? I'm clearly not at ease with this mechanism of slots, widgets...